lib/liveValidation/liveValidator.ts

```diff
@@ -198,7 +198,7 @@ export class LiveValidator {
       container,
       fileRoot: this.options.directory,
       ...this.options,
-      loadSuppression: this.options.loadSuppression ?? Object.keys(allErrorConstants),
+      loadSuppression: this.options.loadSuppression ?? Object.keys(),
```

Comment:
Why do we change default behavior?

Reply:
Unfinished. Should use another way to compute all the errorcode